CHANGELOG.md

```diff
@@ -29,6 +28,8 @@ This section is for maintaining a changelog for all breaking changes for the cli
 ## [Unreleased 2.x]
 
 ### Added
+- Document HTTP/2 support ([#330](https://github.com/opensearch-project/opensearch-java/pull/330))
```

Comment:
Please move this like back, it is only for 3.0.0, thank you

Reply:
Done